### PR TITLE
Initial frontend setup with React/ESlint/Git-hooks

### PR DESCRIPTION
**Pull request by iqraabdulrauf**
Completed following tasks

1- Create react app
2- Configure eslint - airbnb styleguide
3- Configure Husky as a pre-commit hook so that it lints only staged files

May contain many commits as I was testing husky git hook and eslint various times.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (oakbani) [inline .eslintrc]: always name your branch with your prefix and a meaningful name such as 
iqra/create-react-app

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "extends": [
+      "react-app",
```
Comment (oakbani) [inline .eslintrc]: rename your PR title to some meaningful name.

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      // "no-multiple-empty-lines": 2,
+      "class-methods-use-this": 0,
+      "comma-dangle": ["error", "never"],
+      // "no-console": ["error", { "allow": ["warn", "log"] }],
+      "prettier/prettier": ["error", { "singleQuote": true }],
+      "import/prefer-default-export": "off"
+  
+      
+    }
+  }
```
Comment (oakbani) [inline .eslintrc]: always leave an empty line at the end of file.

File: .eslintrc [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+    "extends": [
+      "react-app",
+      "airbnb",
+      "plugin:jsx-a11y/recommended",
+      "plugin:prettier/recommended"
+    ],
+    "plugins": [
```
Comment (oakbani) [inline .eslintrc]: move all of this into a folder named frontend.
We are going to have a backend and a frontend folder in root.

File: .eslintrc (hunk 3 above)
Comment (oakbani) [reply .eslintrc]: do this for all files. Configure settings in your editor so that it always does it.

**Review by oakbani — CHANGES_REQUESTED** [review]
Good start!. Please check my comments

File: .eslintrc (hunk 2 above)
Comment (iqraaabdulrauf) [reply .eslintrc]: noted

File: .eslintrc (hunk 4 above)
Comment (iqraaabdulrauf) [reply .eslintrc]: sure noted

**Review by oakbani — APPROVED** [review]
Looks good

**Pull request merged**